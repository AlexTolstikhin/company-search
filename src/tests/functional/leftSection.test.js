import React from 'react';
import renderer from 'react-test-renderer';
import LeftSection from '../../components/functional/leftSection';

it('snaps LeftSection with undefined props', () => {
	const tree = renderer
		.create(<LeftSection />)
		.toJSON();
	expect(tree).toMatchSnapshot();
});

it('snaps LeftSection with defined props', () => {
	const tree = renderer
		.create(<LeftSection 
			description='fuf'
			isSmallScreen
			summary='fuf'
			thumbnail='fuf'
			title='fuf'
		/>)
		.toJSON();
	expect(tree).toMatchSnapshot();
});