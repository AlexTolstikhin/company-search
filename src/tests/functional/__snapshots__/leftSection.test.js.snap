// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snaps LeftSection with defined props 1`] = `
<section
  style={
    Object {
      "width": "100%",
    }
  }
>
  <h2>
    Main Info:
  </h2>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    <span
      style={
        Object {
          "alignItems": "center",
          "background": "#c7ede9",
          "borderRadius": "15px",
          "display": "flex",
          "height": "100px",
          "justifyContent": "center",
          "margin": "10px",
          "width": "100%",
        }
      }
    >
      <img
        alt="fuf thumbnail"
        height="60px"
        src="fuf"
      />
    </span>
    <span
      style={
        Object {
          "alignItems": "center",
          "background": "#c7ede9",
          "borderRadius": "15px",
          "display": "flex",
          "height": "100px",
          "justifyContent": "center",
          "margin": "10px",
          "width": "100%",
        }
      }
    >
      <h1
        style={Object {}}
      >
        fuf
      </h1>
    </span>
  </div>
  <div>
    <div
      style={
        Object {
          "background": "#c7ede9",
          "borderRadius": "15px",
          "fontSize": "15px",
          "lineHeight": "20px",
          "margin": "10px",
          "padding": "10px 20px 20px 20px",
          "textAlign": "justify",
        }
      }
    >
      <h2>
        Summary
      </h2>
      <hr />
      <div>
        fuf
      </div>
    </div>
    <div
      style={
        Object {
          "background": "#c7ede9",
          "borderRadius": "15px",
          "fontSize": "15px",
          "lineHeight": "20px",
          "margin": "10px",
          "padding": "10px 20px 20px 20px",
          "textAlign": "justify",
        }
      }
    >
      <h2>
        About
      </h2>
      <hr />
      <div>
        fuf
      </div>
    </div>
  </div>
</section>
`;

exports[`snaps LeftSection with undefined props 1`] = `
<section
  style={
    Object {
      "width": "60%",
    }
  }
>
  <h2>
    Main Info:
  </h2>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    
  </div>
  <div>
    
    
  </div>
</section>
`;
